nsls2forge_utils/tests/test_all_feedstocks.py

```diff
@@ -0,0 +1,54 @@
+import os
+import shutil
+import subprocess
+
+from nsls2forge_utils.all_feedstocks import (
+    get_all_feedstocks_from_github,
+    get_all_feedstocks,
+    all_feedstocks_info
+)
+
+
+def test_all_feedstocks_from_github():
+    names = get_all_feedstocks_from_github()
+    assert names is None
+    names = get_all_feedstocks_from_github(organization='nsls-ii-forge')
+    archived_names = get_all_feedstocks_from_github(organization='nsls-ii-forge',
+                                                    archived=True)
```

Comment:
The variable name is a bit misleading as it hints only the archived feedstocks will be included. Maybe worth to rename the kwarg to something like `include_archived`?